exporters/geneva/cmake/package.cmake

```diff
@@ -0,0 +1,71 @@
+set(CPACK_PACKAGE_DESCRIPTION "OpenTelemetry C++ Geneva Exporter for Metrics")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "OpenTelemetry C++ Geneva Exporter for Metrics - C++ Implementation of OpenTelemetry C++ Exporter to Upload Metics to Geneva")
+set(CPACK_PACKAGE_VENDOR "OpenTelemetry")
+set(CPACK_PACKAGE_CONTACT "OpenTelemetry-cpp")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://opentelemetry.io/")
+set(CMAKE_PROJECT_NAME "opentelemetry-cpp-geneva-metrics")
+set(OPENTELEMETRY_VERSION "1.0.0")

```

Comment:
Could we get the version number from opentelemetry-cpp package (header file)?

Reply:
This is supposed to be version number of current package. I have renamed the variable to reflect it clearly.